### DISCUSSION TITLE
iPadOS16 “Blink” would like to paste from “OtherApp” message

### DISCUSSION THREAD

**Discussion by walk0080** [blinksh/blink#1710] [Support] [answered]
This seems to be a common problem and maybe related to iPad/iOS16 and not Blink, but hoping Blink team are working on a fix with Apple?

Trying to paste text from one app into Blink causes an annoying message pop-up, for example:

“Blink” would like to paste from “Notes” Do you want to allow this?
Don’t Allow Pate
Allow Paste

![8F9B4C4B-F778-47AE-AA0F-38E414FE63DA](https://user-images.githubusercontent.com/22773927/220740102-63a1030c-12f0-4cad-8f8b-50125d1f72eb.jpeg)

**Comment by carloscabanero** [comment, marked as answer]
The behavior you see is actually iPadOS and it was quite controversial. Fortunately Apple introduced a fix that allows you to select the behavior you want on an app by app basis. You can check out the instructions here: 

https://www.macrumors.com/2022/10/17/ios-16-1-paste-from-other-apps-settings/

**Comment by walk0080** [comment]
Think you are missing the link?

Only things I have found are update OS, update App, but no way to disable this.

**Reply by carloscabanero** [reply]
Hehe, sorry about that. Updated with the link.

**Reply by walk0080** [reply]
Fantastic thank you !
